### PR TITLE
refactor: simplify to make seata-common more readable

### PR DESCRIPTION
**Pull request by koonchen**
### Ⅰ. Describe what this PR did

I studied and read the seata source code in the seta-common module and found some optimization points.

### Ⅴ. Special notes for reviews

### CODE REVIEW

**Comment by zjinlei** [comment]
Thanks!
Coming soon 1.0, priority review after 1.0.

File: common/src/main/java/io/seata/common/util/NetUtil.java [hunk 1]
```diff
@@ -164,16 +164,14 @@ private static InetAddress getLocalAddress0() {
...
-                                } catch (Throwable e) {
-                                    LOGGER.warn("Failed to retrieving ip address, " + e.getMessage(), e);
+                        while (addresses.hasMoreElements()) {
+                            try {
+                                InetAddress address = addresses.nextElement();
+                                if (isValidAddress(address)) {
+                                    return address;
                                 }
+                            } catch (Throwable e) {
+                                LOGGER.warn("Failed to retrieving ip address, " + e.getMessage(), e);
```
Comment (zjinlei) [inline common/src/main/java/io/seata/common/util/NetUtil.java]: please user {} for e. getMessage()

File: common/src/main/java/io/seata/common/util/NetUtil.java [hunk 2]
```diff
@@ -164,16 +164,14 @@ private static InetAddress getLocalAddress0() {
                     try {
                         NetworkInterface network = interfaces.nextElement();
                         Enumeration<InetAddress> addresses = network.getInetAddresses();
-                        if (addresses != null) {
```
Comment (zjinlei) [inline common/src/main/java/io/seata/common/util/NetUtil.java]: It is recommended to keep and shield external changes.

Comment (koonchen) [reply common/src/main/java/io/seata/common/util/NetUtil.java]: Ok. I corrected it.

File: common/src/main/java/io/seata/common/util/NetUtil.java (hunk 1 above)
Comment (koonchen) [reply common/src/main/java/io/seata/common/util/NetUtil.java]: Ok. I corrected it.

**Comment by koonchen** [comment]
@slievrly Thank you, can you review this pr when you are free?

**Pull request merged**